com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ModelUtil.java

```diff
@@ -670,48 +670,56 @@ private static void findModelNodes(Node node, List<Node> modelNodes) {
         }
     }
 
+    private static void calcCenterNode(Node node, Aabb aabb) {
+        if (node.model != null) {
+            // As a default, we only count nodes with models, as the user
+            // cannot currently see/use the lights or cameras etc that are present in the scene.
+            aabb.expand(node.world.translation.x, node.world.translation.y, node.world.translation.z);
+        }
```

Comment:
The center realignment is for the users benefit, to make it easier to directly place a model in the scene, where the .model is placed.

So, we only let the models contribute, and not the lights/cameras etc.